JustSaying/JustSayingFluentlyExtensions.cs

```diff
@@ -12,6 +14,8 @@ public static class JustSayingFluentlyExtensions
     {
         public static IMayWantOptionalSettings InRegion(this JustSayingFluentlyLogging logging, string region) => InRegions(logging, region);
 
+        public static IMayWantOptionalSettings InRegions(this JustSayingFluentlyLogging logging, IEnumerable<string> regions) => InRegions(logging, regions.ToArray());
```

Comment:
Yeah, so arrays will pick the `string[]` overload, things like `List<string>` will use the other overload, all works as expected. Unless you mean reversing the overloads, which makes more sense, we'd need to use `.AsEnumerable()` to not create an infinite recursive call.